### PR TITLE
:ambulance: fix db name vulnerability

### PR DESCRIPTION
**Pull request by trojikman**

### CODE REVIEW

File: saas/models/saas_db.py [hunk 1]
```diff
@@ -20,11 +21,17 @@ class SAASDB(models.Model):
         ('done', 'Ready'),
     ], default='draft')
 
+    @api.depends('name')
+    def _compute_safe_name(self):
+        for r in self:
+            if r.name:
+                r.safe_name = r.name.replace(' ', '_').lower()
```
Comment (yelizariev) [inline saas/models/saas_db.py]: Use ``slugify`` instead

**Review by yelizariev — CHANGES_REQUESTED** [review]
I'd prefer to fixing name on saving

File: saas/models/saas_db.py (hunk 1 above)
Comment (trojikman) [reply saas/models/saas_db.py]: Cool

File: saas/doc/changelog.rst [hunk 2]
```diff
@@ -1,3 +1,8 @@
+`1.0.2`
+-------
+
+- **Fix:** fix db name vulnerability
```
Comment (yelizariev) [inline saas/doc/changelog.rst]: What is that?

**Review by yelizariev — COMMENTED** [review]
Consider to clean up the commits

File: saas/doc/changelog.rst (hunk 2 above)
Comment (trojikman) [reply saas/doc/changelog.rst]: Changelog message?

Comment (yelizariev) [reply saas/doc/changelog.rst]: I mean I don't understand what does it fix

Comment (trojikman) [reply saas/doc/changelog.rst]: after the user submits the name for the database, it will be processed so that it can be used in the link

Comment (yelizariev) [reply saas/doc/changelog.rst]: ok, I got the idea, but it doesn't come from your description. Please clarify changelog (e.g. "make deafult db name compatible with new slugify function")

File: saas/demo/saas_template_demo.xml [hunk 3]
```diff
@@ -9,7 +9,7 @@
         <field name="build_post_init">env['res.partner'].create({{'name': '{name}'}})</field>
     </record>
     <record id="saas_template_operator" model="saas.template.operator">
-        <field name="operator_db_name">template_database</field>
+        <field name="operator_db_name">template-database</field>
```
Comment (yelizariev) [inline saas/demo/saas_template_demo.xml]: It must be allowed for template use non-slugified templates

**Review by yelizariev — CHANGES_REQUESTED** [review]

**Review by yelizariev — CHANGES_REQUESTED** [review]
Looks good, but squash commits

**Comment by trojikman** [comment]
> Looks good, but squash commits

Done

**Comment by yelizariev** [comment]
I approve to merge it now

**Comment by yelizariev** [comment]
I approve to merge it now

**Comment by yelizariev** [comment]
I approve to merge it now

**Comment by yelizariev** [comment]
I approve to merge it now

**Comment by yelizariev** [comment]
I approve to merge it now

**Pull request merged**